components/core/Test.py

```diff
@@ -0,0 +1,16 @@
+from UserManager import * 
+import unittest
+username = "admin"
+password = "admin"
+usernamer = "rand"
+passwordr = "pass"
+class Test(unittest.TestCase):
```

Comment:
Let's use function docstrings for each of the test cases

Reply:
Done ..